documentation/decisions/0001-formalism-for-aspect-models.md

```diff
@@ -0,0 +1,128 @@
+# Formalism for the definition of Aspect Models
```

Comment:
Considering I'm a reader of the ADRs trying to understand the current state of the software and "how did it come to this", maybe it can help me if I don't have the (often abstract) title of the thing to be decided, but also the (often more concrete) short title of the decision that was made. Does it make sense to add that?

E.g.  "Formalism for the definition of Aspect Models: RDF with ttl"